kafka/README.md

```diff
@@ -0,0 +1,9 @@
+# Setting up kafka
+Install docker-compose and use docker-compose up command in the root directory
```

Comment:
Can you provide step-by-step instruction (with particular CLI commands) for docker newcomers? I mean, I have docker installed and what is next? Where should i get the image, how should i use compose etc. Will be great if you do this

Reply:
The only needed CLI command is "docker-compose up", which is already mentioned.

Images will be pulled from the docker hub automatically.

Reply:
Could you just highlight this part about docker-compose up with <code> tag